addons/test_mail/tests/test_ir_actions.py

```diff
@@ -118,6 +120,61 @@ def test_action_message_post(self):
                          '2 new messages produced')
         self.assertEqual(self.test_partner.message_partner_ids, self.test_partner)
 
+    def test_action_discuss_post(self):
```

Comment:
discuss tests should not be in `test_mail`, all models are available in mail so they can be directly in a discuss subfolder in tests of mail